tokio/src/io/driver/mod.rs

```diff
@@ -274,7 +273,7 @@ cfg_not_rt! {
         /// This function panics if there is no current reactor set, or if the `rt`
         /// feature flag is not enabled.
         pub(super) fn current() -> Self {
-            panic!("there is no reactor running, must be called from the context of Tokio runtime with `rt` enabled.")
+            panic!("feature `rt` must be enabled on Tokio 1.x")
```

Comment:
Honestly, I think using the standardized error message here too would be fine. I mean sure, you need to enable `rt` to fix this error, but the root of the problem is that there is no runtime.

Reply:
@Darksonn OK, I might do that. I kind of see it the other way around though I guess, i.e. that the root of problem is that the "rt" feature is disabled.

Reply:
I mean, if you enable the `rt` feature, you're just going to get the other error about a runtime missing. :woman_shrugging:

Reply:
Yeah.

Reply:
Done.